### PR TITLE
✨Add Country model and endpoint

### PR DESCRIPTION
**Pull request by sofokli1**
Needed for #1610

Adds Country model, api endpoints and tests

Squash whenever

### CODE REVIEW

File: Toggl.Ultrawave.Tests.Integration/CountriesApiTests.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using System.Threading.Tasks;
+using FluentAssertions;
+using Toggl.Multivac.Models;
+using Toggl.Ultrawave.Exceptions;
+using Toggl.Ultrawave.Models;
+using Toggl.Ultrawave.Network;
+using Toggl.Ultrawave.Tests.Integration.BaseTests;
+using Toggl.Ultrawave.Tests.Integration.Helper;
+using Xunit;
+using ThreadingTask = System.Threading.Tasks.Task;
```
Comment (arturs105) [inline Toggl.Ultrawave.Tests.Integration/CountriesApiTests.cs]: Some of these `using`s are not necessary.

File: Toggl.Ultrawave.Tests/Models/CountryTests.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+using Toggl.Ultrawave.Models;
+using Toggl.Ultrawave.Serialization;
+using Xunit;
+
+namespace Toggl.Ultrawave.Tests.Models
+{
+    public sealed class CountryTests
+    {
+        private string validJson
+        => "{\"id\":235,\"name\":\"United States\",\"vat_applicable\":false,\"vat_regex\":null,\"vat_percentage\":null,\"country_code\":\"US\"}";
```
Comment (arturs105) [inline Toggl.Ultrawave.Tests/Models/CountryTests.cs]: Indent by 4 more spaces

File: Toggl.Ultrawave/ApiClients/CountriesApi.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        private readonly CountryEndpoints endPoints;
+
+        public CountriesApi(Endpoints endPoints, IApiClient apiClient, IJsonSerializer serializer, Credentials credentials)
+            : base(apiClient, serializer, credentials, endPoints.LoggedIn)
+        {
+            this.endPoints = endPoints.Countries;
+        }
+
+        public IObservable<List<ICountry>> GetAll()
+        => CreateListObservable<Country, ICountry>(endPoints.Get, AuthHeader);
```
Comment (arturs105) [inline Toggl.Ultrawave/ApiClients/CountriesApi.cs]: Needs more spaces

File: Toggl.Ultrawave.Tests.Integration/CountriesApiTests.cs (hunk 1 above)
Comment (arturs105) [reply Toggl.Ultrawave.Tests.Integration/CountriesApiTests.cs]: You can quickly remove them with `option` + `enter`

File: Toggl.Ultrawave/Network/Endpoints/CountryEndpoints.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    internal struct CountryEndpoints
+    {
+        private readonly Uri baseUrl;
+
+        public CountryEndpoints(Uri baseUrl)
+        {
+            this.baseUrl = baseUrl;
+        }
+
+        public Endpoint Get => Endpoint.Get(baseUrl, "countries");
```
Comment (arturs105) [inline Toggl.Ultrawave/Network/Endpoints/CountryEndpoints.cs]: Please add this endpoint to the list in `docs/endpoints.md`

Comment (sofokli1) [reply Toggl.Ultrawave/Network/Endpoints/CountryEndpoints.cs]: 👍

**Pull request merged**